src/transformers/models/big_bird/configuration_big_bird.py

```diff
@@ -107,7 +107,7 @@ def __init__(
         num_hidden_layers=12,
         num_attention_heads=12,
         intermediate_size=3072,
-        hidden_act="gelu_fast",
+        hidden_act="gelu_new",
```

Comment:
I don't really think we can do this since, it'll change the behavior of all PyTorch BigBird models

Reply:
`gelu_fast` & `gelu_new` looks same to me in pytorch. Let me know if I am wrong in that.

Reply:
Ah I see we don't have `"gelu_fast"` in the flax activations. I think `"gelu_new"` would have been the correct activation anyways since BERT also uses `"gelu_new"` => so I'm fine with changing this one! Could we make the same change for "BigBirdPegasus" then (in the config)? 

Think no model should have `"gelu_fast"` by default actually, so that's a welcoming change :-)

Reply:
Sure will change bigbird Pegasus activation also.